### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by ayebaishmo**
# In this branch 
- Create different modules
- Add mark interaction to tasks
- Add eventListner checkbox  to mark task
- Enabled change of status of task from true to false, reviser. 
- Add local storage to array of tasks

### CODE REVIEW

**Review by BrianSammit — COMMENTED** [review]
# STATUS: INVALID 🟡 
Hi @ayebaishmo, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 😢 

You have submitted a project, that the linters checks are not working on GitHub.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

programming-punching-keyboard-coding-anger-13716030304


# CHEERS AND HAPPY CODING!! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

Invalid Code Review Request **does not count** into the code reviews limit.

File: src/index.js [hunk 1]
```diff
@@ -1,45 +1,57 @@
...
+document.addEventListener('DOMContentLoaded', () => {
+  displayTasks();
+  const checkbox = document.querySelectorAll('.check');
+  checkbox.forEach((element) => {
+    element.addEventListener('change', (e) => {
+      status.changeStatus(e.target, toDoTask);
+      cancelTask.changeStatus(e.target);
+      storeLocal.local(toDoTask);
+    });
+  });
```
Comment (TedLivist) [inline src/index.js:55]: There seem to be some irregularities in the checkbox displays. It does not work properly. Look at the image below
- [ ] For the first image, I checked some tasks as completed and then reloaded the page. After the reload when I click on the checkbox it changes but the strikethrough is gone as evident on the second list item. There should be a strikethrough on the task as long as the box is checked. Kindly fix this behavior.
- [ ] To the second image. When I reload, the completed tasks have expected strikethrough but their checkboxes are not checked. Kindly ensure that checkboxes of completed tasks are correctly checked on page reload.

ayeb-check
ayeb-check-reload

File: src/index.js [hunk 2]
```diff
@@ -1,45 +1,57 @@
 import './style.css';
```
Comment (TedLivist) [inline src/index.js:1]: - [ ] Let's stick to the design as much as possible. There appear to be some issues with your styling. Kindly style your app as provided in the design template.

ayeb-design

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ayebaishmo,

Good job so far!
✔️ You have descriptive PR
✔️ Your code is nicely modularized
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Your repo contains the `node_modules` folder. This should be correctly ignored by git. Kindly ensure that this folder is not committed to your repo.
ayeb

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HillaryOkello — CHANGES_REQUESTED** [review]
# Hi @ayebaishmo , 

## To highlight
✅  You have descriptive PR
✅  Descriptive Readme file
✅  Your code is nicely modularized

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] You still have the node_modules folder in your GitHub repo, to remove it please delete it then push to GitHub then do npm install and push again to Github with the node_modules in .gitignore file.
interactive


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -6,58 +6,31 @@ env:
   FORCE_COLOR: 1
```
Comment (carlos-ssh) [inline .github/workflows/linters.yml:6]: # 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- ⏏︎ Kindly add your `node_modules` to your `.gitignore`.
Screen Shot 2021-12-03 at 17 44 39

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @ayebaishmo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**